diff --git a/minimatch.js b/minimatch.js
index e50ed16..2f6df23 100644
--- a/minimatch.js
+++ b/minimatch.js
@@ -7,7 +7,7 @@ var path = (function () { try { return require('path') } catch (e) {}}()) || {
 minimatch.sep = path.sep
 
 var GLOBSTAR = minimatch.GLOBSTAR = Minimatch.GLOBSTAR = {}
-var expand = require('brace-expansion')
+var expand = require('brace-expansion').expand
 
 var plTypes = {
   '!': { open: '(?:(?!(?:', close: '))[^/]*?)'},
